.eslintrc.json

```diff
@@ -30,6 +30,14 @@
         "ecmaVersion": 2019,
         "sourceType": "module",
         "allowImportExportEverywhere": true
+      },
+      "rules": {
+        "svelte/valid-compile": [
+          "error",
+          {
+            "ignoreWarnings": true
+          }
+        ]
```

Comment:
How come this is required?

Reply:
That was needed on the initial load, due eslint/svelte plugins conflicts. I just confirmed this is not needed anymore! :)